examples/sample-pub/publisher.js

```diff
@@ -144,6 +144,36 @@ function startFlowAuto() {
     return;
   }
 
+  if (flow == 'metering') {
+    whenReady((subsciptions) => {
+      subsciptions
+        .getEntitlements({
+          metering: {
+            userState: {
+              id:
+                'user5901e3f7a7fc5767b6acbbbaa927d36f5901e3f7a7fc5767b6acbbbaa927',
+              attributes: [
+                {
+                  name: 'registered_user',
+                  timestamp: (Date.now() / 1000) | 0,
```

Comment:
What's with the  `| 0`

Reply:
It's a bit faster than using `Math.floor`. I updated the code to use `Math.floor` instead for readability. This will also help the example be more future proof, since the bitwise OR 0 trick only works up to values of 2^31−1. :hourglass_flowing_sand: